app/config/config.yml

```diff
@@ -117,9 +117,6 @@ swiftmailer:
 csa_guzzle:
     profiler:
         enabled: "%kernel.debug%"
-    cache:
-        enabled: true
-        adapter: guzzle.cache
```

Comment:
Is this change related?

Reply:
yes, the new version of the bundle doesn't accept that configuration.

I haven't investigated so far on this, mostly because I think we will use the circuit breaker to query the list of modules and use the integrated cache of Symfony instead: am I wrong?